src/node/biquad_filter.rs

```diff
@@ -609,12 +609,13 @@ impl AudioProcessor for BiquadFilterRenderer {
         let mut current_gain = gain[0];
         let mut current_coefs = Coefficients::default();
 
+        // @todo - optimization only compute coefs once if all params have length == 1
         coefs_list
             .iter_mut()
-            .zip(frequency.iter())
-            .zip(detune.iter())
-            .zip(q.iter())
-            .zip(gain.iter())
+            .zip(frequency.iter().cycle().take(RENDER_QUANTUM_SIZE))
```

Comment:
`take` is not necessary after `cycle` because `zip` will stop after the shortest iterator (which is `coefs_list`)
This goes for all usage of `take` in this PR!

Reply:
yup you are right, was a bit too much defensive, I removed them all